sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -684,7 +689,14 @@ export class MessageSession extends LinkEntity<Receiver> {
           }
           return;
         } finally {
-          this.receiverHelper.addCredit(1);
+          try {
+            this.receiverHelper.addCredit(1);
+          } catch (err) {
+            logger.logError(
+              err,
+              `[${this.logPrefix}] Failed to add credit after receiving message`
```

Comment:
Shouldnt we throw the SessionLockLost from here as well? If addCredit does throw an error here,  there is no forever loop, and so the user should create a new session receiver.

Reply:
Done!